nevergrad/benchmark/experiments.py

```diff
@@ -1194,19 +1200,31 @@ def image_multi_similarity(seed: tp.Optional[int] = None, cross_valid: bool=Fals
                     yield xp
 
 
+@registry.register
+def image_multi_similarity_pgan(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of image_similarity, using PGan as a representation."""
+    return image_multi_similarity(seed, with_pgan=True)
+
+
 @registry.register
 def image_multi_similarity_cv(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     """Counterpart of image_multi_similarity with cross-validation."""
     return image_multi_similarity(seed, cross_valid=True)
 
 
 @registry.register
-def image_quality_proxy(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+def image_multi_similarity_pgan_cv(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of image_multi_similarity with cross-validation."""
```

Comment:
Just another typo : Counterpart of image_multi_similarity with cross-validation, using PGan as a representation.

Reply:
Done.